core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -777,6 +782,15 @@ oppia.directive('conversationSkin', [
             siteAnalyticsService.registerVisitOppiaFromIframeEvent(
               $scope.explorationId);
           };
+
+          $scope.interactionAnswerIsValid = true;
+          $scope.setInteractionAnswerValidity = function(answerValidity) {
+            $scope.interactionAnswerIsValid = answerValidity;
+          }
```

Comment:
Missing semicolon.

Perhaps add a note that this is needed here in order to be available for the submit button in the navbar? I think we should be quite liberal in documenting the reasoning behind all this, since it's introducing temporary tangle to the codebase.

Reply:
Done.